app/controllers/UserAnalysis.scala

```diff
@@ -31,13 +31,23 @@ final class UserAnalysis(
       case _ => load("", Standard)
 
   def load(urlFen: String, variant: Variant) = Open:
-    val decodedFen: Option[Fen.Full] = lila.common.String
+    val inputFen: Option[Fen.Full] = lila.common.String
       .decodeUriPath(urlFen)
       .filter(_.trim.nonEmpty)
       .orElse(get("fen"))
       .map(Fen.Full.clean)
-    val pov         = makePov(decodedFen, variant)
-    val orientation = get("color").flatMap(Color.fromName) | pov.color
+    val chess960PositionNum: Option[Int] =
+      variant.chess960
+        .so(
+          get("position")
+            .flatMap(_.toIntOption)
+            .orElse(
+              Chess960.positionNumber(inputFen | variant.initialFen)
+            ) // no input fen or num defaults to standard start position
+        )
+    val decodedFen: Option[Fen.Full] = chess960PositionNum.flatMap(Chess960.positionToFen).orElse(inputFen)
```

Comment:
To allow `/chess/960?position={num}` or `/chess960/fen` (which could be a starting fen). But it doesn't feel right to me